src/main/java/com/mythsoftware/rest/resource/UserResource.java

```diff
@@ -0,0 +1,40 @@
+package com.mythsoftware.rest.resource;
+
+import com.mythsoftware.rest.model.User;
+import com.mythsoftware.rest.service.UserService;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ */
+@Named
+@Path("/user")
+public class UserResource {
+
+    @Inject
+    private UserService userService;
+
+    @GET
+    @Path("/getUsers")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getUsers() {
+        List<User> users = userService.getUsers();
+        return Response.ok(users).build();
+    }
+
+    @GET
+    @Path("/getUserById/{id}")
```

Comment:
Wouldn't that depend on if you are getting back a single user or a list of users?  Also, the way the code is currently, does that expose the endpoint like this: "/user/getUsers"?